src/planscape/collaboration/services.py

```diff
@@ -26,6 +26,21 @@ def validate_ownership(user, instance):
             return False
 
 
+def get_role(user, instance):
+    is_owner = validate_ownership(user, instance)
+    if is_owner:
+        return Role.OWNER
```

Comment:
If we are doing a method for getting the role, it would be helpful to return "Creator" here instead?. Basically it will save us for doing basically validate_ownership on the FE.

Reply:
Absolutely. Anything else that would be helpful to include to reduce calls from the FE?